front/app/component-library/components/CheckboxWithLabel/index.tsx

```diff
@@ -41,6 +41,7 @@ const CheckboxWithLabel = ({
       alignItems="center"
       onClick={handleLabelClick}
       data-testid={dataTestId || `${testEnv('check-mark-label')}`}
+      style={{ cursor: 'pointer' }}
```

Comment:
Now the label is also gets this style (better UX since you can toggle the checkbox by clicking the label)